stories/Colors/Colors.jsx

```diff
@@ -6,11 +6,15 @@ import propTypes from 'prop-types';
  */
 function Swatch(props) {
```

Comment:
I would keep the same layout for the swatches that we had before where we pull the colors from the theme js and process them with chroma-js. If we are adding shades and variations we should come up with a new format maybe similar to [Shopify's Polaris](https://polaris.shopify.com/design/colors) and include the wcga ratings for each. 

This is where extraction the theme vars into js comes in handy

Reply:
Seems out of scope of this pr. We can do that in a latter one.